src/rust/ensogl/src/display/scene.rs

```diff
@@ -340,8 +340,8 @@ impl Mouse {
 
         let on_move = mouse_manager.on_move.add(f!([frp,shape,position,last_position](event:&mouse::OnMove) {
             let pixel_ratio  = shape.pixel_ratio() as i32;
-            let screen_x     = event.offset_x();
-            let screen_y     = event.offset_y();
+            let screen_x     = event.client_x();
+            let screen_y     = event.client_y();
```

Comment:
Why there was `offset_x` and `y` here??!!! It was causing reflow: https://gist.github.com/paulirish/5d52fb081b3570c81e3a

I was tracking why the reflow is caused for months already and missed that. :(